analysis-core/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+PATH=$PATH:/var/lib/neo4j/bin/
+
+# Start Neo4j
+cd /var/lib/neo4j
+
+chmod a+rw -R /data/databases/reactome.graphdb
+
+bash /docker-entrypoint.sh neo4j &
+
+echo "Waiting for Neo4j..."
+bash /wait-for.sh localhost:7687 -t 90 && bash /wait-for.sh localhost:7474 -t 90
+
+time java -jar /applications/analysis-core-jar-with-dependencies.jar -h localhost -p 7474 -u $NEO4J_USER -k $NEO4J_PASSWORD -o /output/analysis.bin -v
```

Comment:
One consideration would be to remove the '-v' flag from running this. Considering the amount of logging output being generated when we are building the docker images (if that is all this is used for), I think it hurts more than it helps because they are almost entirely just update logs, and there is a lot of updates!

Reply:
Maybe there needs to be a happy medium between SO MUCH OUTPUT and nothing. I think I added the `-v` so that if there was any error, there would be some context to it. But if there's too much output, we can take that out.

Reply:
Yea, your right. The `verbose` switch is the only way to get detailed logging of the analysis-core -- the default logs produced are only for STDERR. It would be nice if the stdout logs were saved as well, and then the jenkins view would only have the 'progress' (and errors) of the docker build, but not the INFO. Ah well 🤷‍♂️